test/unit/alignment/matrix/detail/trace_iterator_test.cpp

```diff
@@ -7,17 +7,19 @@
 
 #include <gtest/gtest.h>
 
+#include <seqan3/std/iterator>
 #include <type_traits>
 #include <vector>
 
 #include <seqan3/alignment/matrix/detail/trace_iterator.hpp>
 #include <seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp>
 #include <seqan3/alignment/matrix/trace_directions.hpp>
 #include <seqan3/range/views/to.hpp>
-#include <seqan3/std/iterator>
 
 #include "../../../range/iterator_test_template.hpp"
 
+using seqan3::operator|;
```

Comment:
why did the "|" Operator work before you changed this file ? Or, if it did work before, why did you add it here ?

Reply:
hope I can explain this correctly:
It's an issue with the enum_bitwise_operators, that the ADL is not working and so the operators are not visible. We discussed this in another PR: https://github.com/seqan/seqan3/pull/1565 and added a discussion Card: https://github.com/orgs/seqan/projects/4#card-32890520

Reply:
thanks !:)